Startup.cs

```diff
@@ -62,7 +66,7 @@ public void ConfigureServices(IServiceCollection services)
                 // options.TokenValidationParameters.ValidIssuers collection
 
                 // Response type
-                options.ResponseType = "code";
+                options.ResponseType = "id_token code";
```

Comment:
id_token reponse type is needed if we want a session cache, so that we have claims about the user identity